smurfs/src/components/EditSmurf/EditSmurfForm.js

```diff
@@ -0,0 +1,45 @@
+import React, { useContext } from 'react'
+import { Divider, Segment, Grid } from 'semantic-ui-react'
+
+import SearchBox from './SearchBox';
+import EditBox from './EditBox';
+import { SmurfContext } from '../../contexts/index';
+import { useForm } from '../../hooks/useForm';
+
+const EditSmurfForm = () => {
+
+  const [smurfs] = useContext(SmurfContext)
+
+  const [values, handleChanges, resetForm, setValues] = useForm({})
+
+  const submitSmurfForEdit = smurfForEdit => {
+    setValues(smurfs.filter(smurf => smurf.name === smurfForEdit)[0])
+  }
```

Comment:
This breaks if you search by a smurf that doesn't exist. Something like:

```
const submitSmurfForEdit = smurfForEdit => {
    const searchedSmurf = smurfs.filter(smurf => smurf.name === smurfForEdit)[0]
    setValues(searchedSmurf ? searchedSmurf : {})
  }
```

would work, plus a message saying "No smurf found" or something.